stubs/numpy/__init__.pyi

```diff
@@ -0,0 +1,498 @@
+"""
+Numpy's mypy stub. Only type declarations for ndarray, the scalar hierarchy and array creation
+methods are provided.
+"""
+
+# This file is taken from https://github.com/machinalis/mypy-data/tree/master/numpy-mypy/numpy
```

Comment:
Question: Can I use this code or are there licensing issues?

Reply:
cc @dzhulgakov for licensing issues

Reply:
BSD and MIT are very close. As long as we maintain reference to the source it's fine. So it's ok to land it